src/test/java/it/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMIT.java

```diff
@@ -4,15 +4,23 @@
 import com.atlassian.bitbucket.jenkins.internal.config.BitbucketPluginConfiguration;
 import com.atlassian.bitbucket.jenkins.internal.fixture.BitbucketJenkinsRule;
 import com.atlassian.bitbucket.jenkins.internal.scm.BitbucketSCM;
-import com.google.common.collect.ImmutableList;
+import com.google.common.collect.Iterables;
```

Comment:
mmmmmm.. This is coming in from a transitive dependency. Maybe we could not use it?
If we insist on using a decade old guava we should at the very least put it in our pom.xml

Reply:
Fair point, the reason I used it is because it's a test dependency